integrationtest/src/test/resources/springTest/src/test/java/org/mapstruct/itest/spring/SpringBasedMapperTest.java

```diff
@@ -24,6 +24,7 @@
 import org.junit.runner.RunWith;
```

Comment:
This file (and some others) has its file mode changed. I don't think that's intentional?